.github/workflows/build-citus-nightlies.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and publish citus nightly package
+
+env:
+  MAIN_BRANCH: "all-citus"
+  PACKAGE_CLOUD_REPO_NAME: "citusdata/community-nightlies"
+  PACKAGE_CLOUD_API_TOKEN: "${{ secrets.PACKAGE_CLOUD_API_TOKEN }}"
+  GH_TOKEN: "${{ secrets.GH_TOKEN }}"
+  PACKAGING_SECRET_KEY: "${{ secrets.PACKAGING_SECRET_KEY }}"
+  PACKAGING_PASSPHRASE: "${{ secrets.PACKAGING_PASSPHRASE }}"
+
+on:
+  push:
+    branches:
+      - "**"
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - el/7
+          - el/8
+          - ol/7
+          - debian/stretch
+          - debian/buster
+          - ubuntu/xenial
+          - ubuntu/bionic
+          - ubuntu/focal
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Clone tools branch
+        run: git clone -b pc_invalid_repo_name --depth=1  https://github.com/citusdata/tools.git tools
+
+      - name: Clone all-citus branch
+        run: git clone -b all-citus --depth=1  https://github.com/citusdata/packaging.git packaging
```

Comment:
what about enterprise ?

Reply:
I can add matrix structure and add enterprise in it but I prefer another PR to make nightly quickly work

Reply:
Please open an issue to make sure we do not forget about enterprise nightly images